page-supporters.php

```diff
@@ -69,6 +69,14 @@
                                      'id'=>'14','linkedin'=>'https://www.linkedin.com/company/edgescan-com',
                                      'twitter'=>'https://twitter.com/edgescan'],
 
+                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/bullet.png', 'name'=>'Bullet', 'jobDescription'=>'Bullet provide payroll software to ensure this accounting for salaries and employee tax is managed in an accurate and efficient manner.',
```

Comment:
"to ensure **this** accounting" Typo? Probably just meant to be "the"? /cc @rosalanghammer